paasta_tools/cli/cmds/status.py

```diff
@@ -2296,6 +2312,27 @@ def _use_new_paasta_status(args, system_paasta_config) -> bool:
             return True
 
 
+def _print_flink_jobs_exceptions_from_api(jobs, cr_name, cluster):
+    with concurrent.futures.ThreadPoolExecutor(
```

Comment:
should we be using asyncio Futures, rather than a threadpool (and worrying about the number of os threads) here?

Reply:
Honestly, I am actually not an expert here, I tried to see what other functions are doing and found that one of the functions in the same file is using threadPoolExecutor 
https://github.com/Yelp/paasta/blob/aff3057aab9663ac1ee79ff784b74b9af2fc1233/paasta_tools/cli/cmds/status.py#L2116

Reply:
@EvanKrall do you have an opinion here? I know you recently moved mark for deployment to use asyncio futures.

Reply:
@Rob-Johnson @EvanKrall Any updates? 
This became a blocker to some rollout plans we had.

Reply:
@AhmedSoliman7 - it was labor day in the US yesterday. If you need to merge urgently then go for it, otherwise @EvanKrall should respond in PST working hours

Reply:
Is it bad to go for it and test the feature and then iterate back on this matter? 

I need to merge and prioritize testing for now, WDYT?

Reply:
as this is in cli/cmds, it's a code that runs on your box, not part of paasta api, so this should be testable by just running `paasta status` from your dev checkout inside a venv. am i missing something?

Reply:
yeah, lets merge @AhmedSoliman7